packages/blob-storage-manager/src/env.ts

```diff
@@ -12,9 +13,28 @@ export const env = createEnv({
     GOOGLE_STORAGE_ENABLED: booleanSchema.default("false"),
     POSTGRES_STORAGE_ENABLED: booleanSchema.default("true"),
     SWARM_STORAGE_ENABLED: booleanSchema.default("false"),
+    PRINT_CONFIG: booleanSchema.default("false"),
```

Comment:
Same, I don't see the point of having it

Reply:
Yeah you're right. I'll rollback this.